user_guide_src/source/incoming/routing.rst

```diff
@@ -445,8 +445,17 @@ be used when the first parameter is a language string:
 
 .. _routing-priority:
 
-Route processing queue
-----------------------
+Route Priority
+**************
+
+Routes are registered in the routing table in the order in which they are defined, and the first matching route is executed.
+
+.. note:: If exactly the same route (the URI path) is defined more than once, only the route defined first is registered.
+
+You can check the routing table by the :ref:`spark routes <spark-routes>` command.
```

Comment:
I'm not sure if `handlers` is the correct term for the controller actions. Kindly revise if needed.

```suggestion
Routes are registered in the routing table in the order in which they are defined. This means that when a URI is accessed, the first matching route will be executed.

.. note:: If a route (the URI path) is defined more than once with different handlers, only the first defined route is registered.

You can check registered routes in the routing table by running the :ref:`spark routes <spark-routes>` command.
```

Reply:
`handlers` is the correct term. We use it in the `spark routes`.

```
+--------+--------+------------------------------------------+----------------+---------------+
| Method | Route  | Handler                                  | Before Filters | After Filters |
+--------+--------+------------------------------------------+----------------+---------------+
| GET    | /      | \App\Controllers\Home::index             |                | toolbar       |
| CLI    | ci(.*) | \CodeIgniter\CLI\CommandRunner::index/$1 |                |               |
+--------+--------+------------------------------------------+----------------+---------------+
```